lib/src/panel.dart

```diff
@@ -1078,9 +1078,7 @@ class _SlidingPanelState extends State<SlidingPanel> with TickerProviderStateMix
           Positioned.fill(top: _getParallaxSlideAmount(this), child: widget.content.bodyContent),
 
         if (widget.backdropConfig.enabled)
-          _backdropShadow
-        else
-          Container(),
```

Comment:
Thanks. But, removing this won't work. You can check yourself by running the given example: 
"Changing panel's height runtime", where we don't use `backdrop`.

Reply:
If you using the `bodyContent` in SlidingPanel, It works well.
But if you using the SlidingPanel in a stack view. It would mask all the content below the section.
I update the demo `Changing panel's height runtime` to below,  You could verify it.

Reply:
In the attched demo file, all the button can not been clicked. I think the root case is that: all the `_content` was been wrapped by `Material` widget.  But the fastest way to fix that is just remove the empty `Container`.